exporter/sentryexporter/sentry_exporter.go

```diff
@@ -444,7 +443,7 @@ func CreateSentryExporter(config *Config, set component.ExporterCreateSettings)
 			allEventsFlushed := transport.Flush(ctx)
 
 			if !allEventsFlushed {
-				log.Print("Could not flush all events, reached timeout")
+				set.Logger.Warn("Could not flush all events, reached timeout")
```

Comment:
Updated this due to the amount of spam it produced throughout testing.

Reply:
cc @AbhiPrasad